tests/wpt/web-platform-tests/custom-elements/pseudo-class-defined.html

```diff
@@ -16,6 +16,7 @@
 const neither = 'rgb(255, 0, 0)';
 const defined = 'rgb(255, 165, 0)';
 const not_defined = 'rgb(0, 0, 255)';
+var iframe = document.getElementById("iframe"); // don't need to test window named getter here
```

Comment:
Maybe `const` for consistency with above?

Reply:
You are correct; I'm just still in a pre-ES6 habit of blindly typing "var" for everything.